tests/Tests/image-iteration.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Resource         ../Resources/ODS.robot
+Resource         ../Resources/Common.robot
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Variables ***
+#{xyz-n}  image  repo_URL  notebook_path
+@{generic-1}  s2i-generic-data-science-notebook  https://github.com/lugi0/minimal-nb-image-test  minimal-nb-image-test/minimal-nb.ipynb
+@{generic-2}  s2i-generic-data-science-notebook  https://github.com/lugi0/clustering-notebook  clustering-notebook/CCFraud-clustering.ipynb
+@{generic-3}  s2i-generic-data-science-notebook  https://github.com/lugi0/clustering-notebook  clustering-notebook/customer-segmentation-k-means-analysis.ipynb
+@{minimal-1}  s2i-minimal-notebook  https://github.com/lugi0/minimal-nb-image-test  minimal-nb-image-test/minimal-nb.ipynb
+@{minimal-2}  s2i-minimal-notebook  https://github.com/lugi0/clustering-notebook  clustering-notebook/CCFraud-clustering.ipynb
+@{minimal-3}  s2i-minimal-notebook  https://github.com/lugi0/clustering-notebook  clustering-notebook/customer-segmentation-k-means-analysis.ipynb
+
+@{WORKLOADS}  ${generic-1}  ${generic-2}  ${generic-3}
+...           ${minimal-1}  ${minimal-2}  ${minimal-3}
+
+
+*** Test Cases ***
+Open ODH Dashboard
+  [Tags]  Sanity
+  Login To ODH Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for ODH Dashboard to Load
+
+Iterative Testing
+  [Template]  Iterative Image Test
+  [Tags]  Sanity
```

Comment:
I think i have missed something here, I was thinking lines: 28 to 33 will be repeated with different tags.

Reply:
There's no need to do that if we take the "list of lists" approach for the image/repo combo.
We CAN still do it if we want to have the different tags, but then the list of lists makes no sense and we need to change it.